viser/_message_api.py

```diff
@@ -341,6 +341,42 @@ def add_mesh_trimesh(
 
         return MeshHandle._make(self, name, wxyz, position, visible)
 
+    def set_popup_image(
```

Comment:
can we combine this with `set_background_image()`? it could be one function with depth as an optional argument

Reply:
I think combining it might be confusing since "background" implies in the back, popup makes it more obvious what depth actually does. I can combine it if you prefer though

Reply:
I merged it, I think it works out better for users